tfhe/src/keycache/mod.rs

```diff
@@ -21,7 +21,7 @@ pub mod utils {
     }
 
     pub trait NamedParam {
-        fn name(&self) -> &'static str;
+        fn name(&self) -> String;
```

Comment:
why ?

Reply:
Some tests (KeySwitchingKey test if I remember correctly )generates parameters at runtime, so parameters for which there is no associated constant, so no associated &static str.

So the NamedParam had to be changed to allow returning some string built at runtime

Reply:
huh, ok, that's too bad, do you remember where this was ?

Reply:
I think this one is an example https://github.com/zama-ai/tfhe-rs/blob/95dcf95e88299910a51f9f8805786327f5e10127/tfhe/src/shortint/key_switching_key/test.rs#L55

Reply:
arf, indeed

we are hitting the problem as the keys now go through the cache ? or is it because we bench them now ?

Reply:
Yes, to speed up coverage all keys are fetched from the keycache, and some of these key have parameters which are made at runtime and do not have as `const ...`

Reply:
is it expected ? or should we have static params down the line ? i.e. is it a workaround at the keyswitching level or is it the normal flow that is expected later ?

Reply:
I think we did that because for some of the parameters of keyswitching, it did not make sense to add the `const ...`